test/UnitTest/Altinn.Platform.Storage.UnitTest.csproj

```diff
@@ -42,6 +42,7 @@
   <PropertyGroup>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NoWarn>$(NoWarn);1591</NoWarn>
+    <UserSecretsId>719eaaa3-a93f-43e9-bc80-86e01719b323</UserSecretsId>
```

Comment:
is this unique to your personal set up or can it be reused by others working with this project? :) Asking cause I don't know.

Reply:
The id is the same for everyone.